packages/menu/src/MenuDescendant/MenuDescendant.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { PropsWithChildren, ReactElement } from 'react';
+import PropTypes from 'prop-types';
+
+import { useDescendant } from '@leafygreen-ui/descendants';
+import { css } from '@leafygreen-ui/emotion';
+import { InputOption } from '@leafygreen-ui/input-option';
+
+import { MenuDescendantsContext, useMenuContext } from '../MenuContext';
+import { menuColor } from '../styles';
+
+export const MenuDescendant = React.forwardRef<
+  HTMLElement,
+  PropsWithChildren<{}>
+>(({ children }, fwdRef) => {
+  const { theme } = useMenuContext();
+  const { ref, id } = useDescendant(MenuDescendantsContext, fwdRef, {});
+  const child = React.Children.only(children) as ReactElement;
+
+  return child ? (
```

Comment:
do we want to force this coercion to an InputOption component? i don't think so - i think instead we expose the hook that registers an element to the Descendant Context, or just clone the element and pass it the ref/id

Reply:
we can probably do this in focusable menuitem directly

Reply:
You're right - we shouldn't use InputOption here, just a div with padding (to solve the focus issues)

Reply:
Also, (not to self) I need to document that the single child of MenuDescendant needs to be focusable